lua/randomat2/events/sv_poker.lua

```diff
@@ -0,0 +1,1136 @@
+--// Logan Christianson
+
+util.AddNetworkString("StartPokerRandomat")
+util.AddNetworkString("StartPokerRandomatCallback")
+util.AddNetworkString("BeginPokerRandomat")
+util.AddNetworkString("NotifyBlinds")
+util.AddNetworkString("DealCards")
+util.AddNetworkString("StartBetting")
+util.AddNetworkString("MakeBet")
+util.AddNetworkString("PlayerFolded")
+util.AddNetworkString("PlayerChecked")
+util.AddNetworkString("PlayerCalled")
+util.AddNetworkString("PlayerRaised")
+util.AddNetworkString("StartDiscard")
+util.AddNetworkString("PlayersFinishedBetting")
+util.AddNetworkString("MakeDiscard")
+util.AddNetworkString("DeclareNoWinner")
+util.AddNetworkString("DeclareWinner")
+util.AddNetworkString("ClosePokerWindow")
+
+util.AddNetworkString("ShareCards")
+util.AddNetworkString("MarkRoundVariant")
+
+--// EVENT properties
+
+local EVENT = {}
+local EVENT_VARIANT = {}
+local EVENT_REF = nil
+
+EVENT.Title = "A Round Of Yogscast Poker"
+EVENT.Description = "Only if the 9 of Diamonds touch!"
+EVENT.ExtDescription = "A round of 5-Card Draw Poker (no Texas Hold 'Em, for my sake), bet with your\nhealth. Up to 7 may play. Any pair, three, or four of a kind containing the 9 of\nDiamonds instantly wins."
+EVENT.id = "poker"
+EVENT.MinPlayers = 2
+EVENT.Type = EVENT_TYPE_DEFAULT
```

Comment:
You don't need to specify this since it's the default

Reply:
But I like specifying